tests/unit/s2n_sike_p503_kat_test.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * Modified from PQCgenKAT_kem.c
+ * Created by Bassham, Lawrence E (Fed) on 8/29/17.
+ * Copyright © 2017 Bassham, Lawrence E (Fed). All rights reserved.
+ */
+
+#include "s2n_test.h"
+#include "s2n_nist_kats.h"
+#include "pq-crypto/sike/sike_p503_kem.h"
+#include "pq-crypto/pq-random.h"
+
+#define RSP_FILE_NAME "kats/sike_p503.kat"
+
+int main(int argc, char **argv, char **envp) {
+    BEGIN_TEST();
+
+    // Flip pq-random over to nist RNG to ensure KAT values match
+    EXPECT_SUCCESS(initialize_pq_crypto_generator(&randombytes));
```

Comment:
Did you look at pq-random.h? Do you think this is a good way to continue flipping between the RNGs? This is really a review of what I came up with and wondering what you think.